lib/http-context.js

```diff
@@ -491,6 +491,60 @@ HttpContext.prototype.respondWithEventStream = function(stream) {
 };
 
 /**
+ * Deciding on the operation of response, function is called inside this.done()
+ */
+
+HttpContext.prototype.resolveReponseOperation = function(accepts) {
+  var res = this.res;
+  var data = this.result;
+  var result = { // default
+    operation : function() { res.json(data); },
```

Comment:
Performance micro-optimization, hopefully not preliminary:

At the moment, a new function must be created for each HttpContext instance because the closure is always different. IIRC, this prevents V8 from optimising the code. Please rework this function to get `res` and `data` from arguments, not the closure, i.e. `function(res, data) { res.json(data); }`. That should allow V8 to optimise it more.

Even better, define these small functions outside of `HttpContext.prototype.resolveResponseOperation`.

```js
HttpContext.prototype.resolveReponseOperation = function(accepts) {
  // ...
  result.sendBody = sendJson;
  // ...
}

function sendJson(res, data) {
  res.json(data);
}
```